lldb/packages/Python/lldbsuite/test/tools/lldb-dap/dap_server.py

```diff
@@ -380,6 +380,17 @@ def wait_for_event(self, filter=None, timeout=None):
             )
         return None
 
+    def wait_for_events(self, events, timeout=None):
+        """Wait for a list of events in `events` in any order.
+        Return the events not hit before the timeout expired"""
+        events = events[:]  # Make a copy to avoid modifying the input
+        while events:
+            event_dict = self.wait_for_event(filter=events, timeout=timeout)
+            if event_dict is None:
```

Comment:
Do we want an explicit check that `event_dict is None`? Or would it be better to have a truth test like

```
if event_dict:
   ...

```